ansible/vars/enikshay/enikshay_public.yml

```diff
@@ -0,0 +1,287 @@
+SITE_HOST: 'enikshay.in'
+#CAS_SITE_HOST: 'cas.commcarehq.org'
+#ENIKSHAY_SITE_HOST: 'enikshay.commcarehq.org'
```

Comment:
@javierwilson i think we did change this to enikshay.in instead of keeping it enikshay.commcarehq.org.
@emord i am unable to open `enikshay.commcarehq.org` in a browser but can ping it though. Not sure why that is happening though.